src/gwproto/types/batched_readings.py

```diff
@@ -0,0 +1,43 @@
+"""Type batched.readings, version 000"""
+
+from typing import List, Literal
+
+from pydantic import (
+    BaseModel,
+    ConfigDict,
+    PositiveInt,
+    model_validator,
+)
+from typing_extensions import Self
+
+from gwproto.property_format import (
+    LeftRightDotStr,
+    UTCMilliseconds,
+    UTCSeconds,
+    UUID4Str,
+)
+from gwproto.types.channel_readings import ChannelReadings
+
+
+class BatchedReadings(BaseModel):
+    FromGNodeAlias: LeftRightDotStr
+    FromGNodeInstanceId: UUID4Str
+    AboutGNodeAlias: LeftRightDotStr
+    SlotStartUnixS: UTCSeconds
+    BatchedTransmissionPeriodS: PositiveInt
+    MessageCreatedMs: UTCMilliseconds
+    ChannelReadingList: List[ChannelReadings]
+    Id: UUID4Str
+    TypeName: Literal["batched.readings"] = "batched.readings"
+    Version: Literal["001"] = "001"
+
+    model_config = ConfigDict(extra="allow")
```

Comment:
I'm not sure we need this for BatchedReadings specifically. Is there a case where we expect this message to extra fields that we can actully do something with? Perhaps because we recieve this from another scada and want to forward it on without losing info? If that's the case, we should probably set this in the base event class. Something to chew on.

Reply:
This will make it so I can add the various lists I took out (FsmActions) etc before finalizing the addition.